packages/config/src/env/main.js

```diff
@@ -55,21 +55,31 @@ const getEnv = async function ({ mode, config, siteInfo, accounts, addons, build
   return fromEntries(env)
 }
 
+// Utility function, if any prop value is undefined or null return an empty object
+const emptyIfUnsetValues = function (obj) {
```

Comment:
Added an helper function to both improve readibility and avoid running into the complexity eslint rule - https://eslint.org/docs/rules/complexity 👀

Reply:
Would it be simpler to call [`removeFalsy()`](https://github.com/netlify/build/blob/master/packages/config/src/utils/remove_falsy.js#L6) on the return value of `getGeneralEnv()` instead?

Reply:
Good call @ehmicky, replaced it with `removeFalsy()` 👍